app/src/main/java/com/pixlee/pixleeandroidsdk/ui/AnalyticsFragment.java

```diff
@@ -264,30 +215,31 @@ public void onClick(View view) {
     boolean widgetVisible = false;
 
     void openWidget() {
-        v_widget_box.setVisibility(View.VISIBLE);
-        String openedWidgetStatus = "openedWidget " +
+        binding.vWidgetBox.setVisibility(View.VISIBLE);
+        String message = "openedWidget " +
                 (album.openedWidget(PXLWidgetType.photowall) ? "success" : "failed");
-        addWidgetStaus(true, openedWidgetStatus);
+        addWidgetStaus(true, message);
 
-        showToast(openedWidgetStatus + "!!\n\nScroll down to fire Widget Visible");
+        showToast(message + "!!\n\nScroll down to fire Widget Visible");
         widgetVisible = false;
         final Rect scrollBounds = new Rect();
-        scroll_widget.getHitRect(scrollBounds);
-        scroll_widget.setOnScrollChangeListener(new NestedScrollView.OnScrollChangeListener() {
+        binding.scrollWidget.getHitRect(scrollBounds);
+        binding.scrollWidget.setOnScrollChangeListener(new NestedScrollView.OnScrollChangeListener() {
             @Override
             public void onScrollChange(NestedScrollView v, int scrollX, int scrollY, int oldScrollX, int oldScrollY) {
-                if (v_widget != null) {
-                    if (v_widget.getLocalVisibleRect(scrollBounds)) {
+                if (binding.vWidget != null) {
+
+                    if (binding.vWidget.getLocalVisibleRect(scrollBounds)) {
+
                         if (!widgetVisible) {
-                            String widgetVisibleStatus = "widgetVisible " +
+                            String message2 = "visibleWidget " +
```

Comment:
the message2 is back? and it doesnt seem to even be used, the line below uses message not message2?

Reply:
This was my mistake. I need to use this so I fixed this.